### ISSUE TITLE
Geometry overlap with HPSEcal4 in new detector with hodoscope

### ISSUE THREAD

**Issue by mholtrop** [JeffersonLab/hps-java#347]
The "alignment enabled ECAL" model that is in HPSEcal4.java needs to have updated parameters. It is not clear how to do this. All the crystals overlap for the new geometry with the hodoscope.

**Comment by JeremyMcCormick** [comment]
Are we planning on using this in any of our compact detectors?

I don't think it is currently used at all (correct me if I'm wrong).

**Comment by JeremyMcCormick** [comment]
As far as I'm aware, we don't use this subdetector class anywhere and don't plan to anytime soon so I'm closing this.

**Comment by mholtrop** [comment]
I don't think that is a good reason to close an issue that was created to not forget this.

**Comment by JeremyMcCormick** [comment]
> I don't think that is a good reason to close an issue that was created to not forget this.

I am pretty sure that this subdetector class will *never* be used in production if it hasn't been by now, so this seems like clutter.  But okay I will leave it here...
